src/introduction-to-unified.md

```diff
@@ -0,0 +1,215 @@
+### unified collective
+
+The unified collective are like-minded organisations to collaboratively work
+on the innovation of content through seamless, interchangeable,
+and extendible tooling.
+
+Depending on what you want to do you will reference different organisations.
+But before getting into the examples, let’s do an introduction round!
+
+#### Processors
+
+*   [remark][remark] — markdown.
+*   [rehype][rehype] — HTML.
+*   [retext][retext] — natural language.
+*   [redot][redot] — Graphviz.
+
+There is also a new markdown parser in the works, [micromark][micromark]!
+
+#### Tools
+
+*   [mdx][mdx] — JSX in Markdown.
+*   [vfile][vfile] — Virtual file format for text processing.
+
+#### Specifications
+
+*   [unist][unist] — Universal Syntax Tree.
+*   [mdast][mdast] — Markdown Abstract Syntax Tree format.
+*   [hast][hast] — HTML Abstract Syntax Tree format.
+*   [nlcst][nlcst] — Natural Language Concrete Syntax Tree format.
+
+* * *
+
+Don’t worry, we’ll get to how all of this comes together in unified.
+But if you are already feeling adventurous; you can go directly to
+[“Using unified”](<>) or [“How to get started with plugins”](<>).
+
+* * *
+
+### How does it all come together
+
+unified can integrate with the file-system through
+[unified-engine][unified-engine].  On top of that, CLI apps can be created with
+[unified-args][unified-args], Gulp plugins with
+[unified-engine-gulp][unified-engine-gulp], and Atom Linters with
+[unified-engine-atom][unified-engine-atom].
+
+But how do these specifications, tools, and processors fit together?
+**Essentially, unified is a three part act**:
+
+1.  **Parse.**  Whether your input is natural text, HTML, or markdown
+    — it needs to be parsed to a universal workable format.
+    The processors (e.g [remark][remark]) are responsible for doing just that.
+    The specifications (e.g [mdast][mdast] for remark) make sure these formats
+    are enforced standards.
+2.  **Transform.**  This is where the magic happens.  Plugins are able to easily
+    hook into this phase and add, change, or remove anything that it’s given.
+    Thanks to unified’s pipeline, you can compose plugins and define the order
+    they run in.
+3.  **Stringify.**  When all plugins have ran, the final step is to take the
+    (adjusted) representation and transform it back to its original format
+    (or to a different format!).
+
+#### Bridging formats
+
+What makes unified unique is that it can continuously switch between formats,
+markdown to HTML for instance, in the same process.
+This allows for even more powerful compositions.
+
+Bridging formats can be done with the following plugins:
+
+*   [remark-rehype][remark-rehype] — markdown to HTML.
+*   [rehype-remark][rehype-remark] — HTML to markdown.
+
+### Use cases
+
+**Whenever you think about processing content — you can think of unified**.
+Unified makes it easier for developers to develop by providing the means to
+quickly iterate, manipulate, and create content.
```

Comment:
This above paragraph is super marketing-y. And it doesn’t really give a use case.

I think instead it should align with what John says in the end: `It [provides] the primitives to build on`.

If you’re going to work with content, typically you look for a package that does exactly that: marked to transform Markdown to HTML. But then you want to add a syntax extensions and you try markdown-it instead. Finally you also want to check spelling. Or some other fancy task. That’s where you discover unified. And where unified becomes useful.